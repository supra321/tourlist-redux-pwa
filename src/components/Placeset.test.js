import React from 'react';
import {shallow} from 'enzyme';
import Placeset from './Placeset';

import Srinagar_Image from '../media/images/SRINAGAR.jpg';
import Srinagar_PDF from '../media/pdf/Srinagar.pdf';

it('Expect to render Placeset Component',()=>{
	const mockPlaces=[
		{
			id: 1,
		    destination: 'Srinagar',
		    state: 'Jammu and Kashmir',
		    pakage: '18 Tours & Activity',
		    pdf: Srinagar_PDF,
		    image: Srinagar_Image
		}
	]
	expect(shallow(<Placeset Placelists={mockPlaces}/>)).toMatchSnapshot();
})
