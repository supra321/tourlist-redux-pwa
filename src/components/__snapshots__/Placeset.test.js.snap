// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Placeset Component 1`] = `ShallowWrapper {}`;
